### ISSUE TITLE
Some references aren't in alphabetical order

### ISSUE THREAD

**Issue by sean-rohan-NOAA** [afsc-gap-products/citations#2]
Based on discussion related to PR #1 

### Issue
Some references in the .bib files are out of order. 

Examples:
https://github.com/afsc-gap-products/citations/blob/3ed62a229350bfeaf468f8c10750ba63e5b59846/cite/bibliography.bib#L933-L957
https://github.com/afsc-gap-products/citations/blob/3ed62a229350bfeaf468f8c10750ba63e5b59846/cite/bibliography.bib#L1053-L1074
https://github.com/afsc-gap-products/citations/blob/3ed62a229350bfeaf468f8c10750ba63e5b59846/cite/bibliography.bib#L1370-L1443
https://github.com/afsc-gap-products/citations/blob/3ed62a229350bfeaf468f8c10750ba63e5b59846/cite/bibliography.bib#L1676-L1747

**Comment by EmilyMarkowitz-NOAA** [comment]
Thanks for these examples! However, I think I must still be missing something obvious here. 

For the functionality of the .bib file, the citation entries don't have to be in any particular order for the computer to read and apply them. But it can be nice for reading through the file, as a human. For reading through, are you suggesting they are out of author- or title- alphabetical order? Probably something else since those ideas don't match with all of the examples you posted. Happy to move citations around for reading ease. 

Or, are you suggesting the authors in the citation (e.g., on the `author = {Wilderbuer, T. K. and Nichol, D. G. and Ianelli, J.},` line) are out of order? There, the first author is the first author of the paper and the second author listed is the second author, etc.

**Comment by sean-rohan-NOAA** [comment]
Yep, totally agree that the entries in the .bib file do not need to be in
alphabetical order to render in alphabetical order. No, it's not about the
authors in the entries being in the wrong order. This is solely about
putting individual entries in alphabetical order for the sake of human
readability, which I think would be helpful if the .bib file grows over
time.

Alternatively, I could create a new .bib file for the ESRs for the
citations repo. The entries I copied over were from the fish condition ESR,
where we've maintained our own .bib and .csl files. As we standardize ESR
products, I'd like to switch to using the citations repo to manage
citations across multiple products, including the ESR, as shown in the code
examples on the citations repo README.
